packages/flutter/lib/src/material/search.dart

```diff
@@ -188,16 +188,19 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.actions], the intended use for the return value of this method.
   List<Widget> buildActions(BuildContext context);
 
-  /// The theme used to style the [AppBar].
+  /// The theme used to configure the search page.
+  ///
+  /// The returned [ThemeData] will be used to wrap the entire search page,
+  /// so it can be used to configure any of its components with the appropriate
+  /// theme properties.
   ///
   /// By default, a white theme is used.
```

Comment:
I suppose we could elaborate "white theme" a little bit.  For example it's a theme where the background is white and the foreground is black.

Reply:
Just added a better description for the default.